pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1220,41 +1220,58 @@ func Convert_v1_Firmware_To_related_apis(vmi *v1.VirtualMachineInstance, domain
 		domain.Spec.OS.KernelArgs = firmware.KernelBoot.KernelArgs
 	}
 
-	if firmware.ACPI != nil {
-		path, err := getSlicMountedPath(vmi.Spec.Volumes, firmware.ACPI.SlicNameRef)
-		if err != nil {
-			log.Log.Object(vmi).Warningf("Failed to get supported path for Volume: %s", firmware.ACPI.SlicNameRef)
-			return err
-		}
+	if err := Convert_v1_Firmware_ACPI_To_related_apis(firmware, domain, vmi.Spec.Volumes); err != nil {
+		return err
+	}
 
-		domain.Spec.OS.ACPI = &api.OSACPI{
-			Table: api.ACPITable{
-				Type: "slic",
-				Path: path,
-			},
-		}
+	return nil
+}
+
+func Convert_v1_Firmware_ACPI_To_related_apis(firmware *v1.Firmware, domain *api.Domain, volumes []v1.Volume) error {
+	if firmware.ACPI == nil {
+		return nil
+	}
+
+	if firmware.ACPI.SlicNameRef == "" && firmware.ACPI.MsdmNameRef == "" {
+		return fmt.Errorf("No ACPI tables were set. Expecting at least one.")
```

Comment:
Should we treat that as an error or better return without doing anything?

Reply:
ACK

Reply:
Maybe I am missing something, but I cannot see how we avoid that `firmware.ACPI.SlicNameRef` and `firmware.ACPI.MsdmNameRef` cannot be empty at the same time.
From the admitter:
```
func validateACPIRef(field *k8sfield.Path, nameRef string, volumes []v1.Volume, fieldName string) []metav1.StatusCause {
	var causes []metav1.StatusCause
	if nameRef == "" {
		return causes
	}
	...
```
We say everything is fine if nameRef is emty (for both slic and mdsm)